bitnami/metrics-server/README.md

```diff
@@ -52,6 +52,7 @@ The following tables lists the configurable parameters of the Metrics Server cha
 | `image.repository`       | Metrics Server image name                                                   | `bitnami/metrics-server`               |
 | `image.tag`              | Metrics Server image tag                                                    | `{VERSION}`                            |
 | `image.pullPolicy`       | Metrics Server image pull policy                                            | `Always`                               |
+| `image.imagePullSecrets` | Reference to secrets to be used when pulling images                         | []                                 |
```

Comment:
As we are under the `image` level, `imagePullSecrets` sounds redundant, can you replace it with `pullSecrets`?

```suggestion
| `image.pullSecrets`                  | Specify image pull secrets                                 | `nil`                                         |
```

Reply:
well... I never saw the abreviation in other charts.. .Not sure it is very clear